src/main/scala/extracells/api/IStorageCellVoid.java

```diff
@@ -0,0 +1,32 @@
+package extracells.api;
+
+
+import appeng.api.storage.ICellWorkbenchItem;
+import appeng.api.storage.data.IAEItemStack;
+import net.minecraft.item.ItemStack;
+
+import javax.annotation.Nullable;
+
+//Copy of IStorageCell
+public interface IStorageCellVoid extends ICellWorkbenchItem
```

Comment:
Why does it need a separate interface? If you need an indicator, just inherit an empty interface from IStorageCell

Reply:
I tried this but the CellHandler wants to return the basic cell inventory and I couldn't figure out a way to fix that without changing AE2 code.

I think AE2 just doesn't support multiple cell handlers that return the same storage cell interface.